composer.json

```diff
@@ -48,7 +48,7 @@
         "laravel/tinker": "~2.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.4|^9.3.3",
+        "phpunit/phpunit": "^8.5.12|^9.3.3",
```

Comment:
Doesn't this mean that the dependency is now locked to 8.5.* where * >= 12?

Reply:
Constraint will be satisfied by versions matching >=8.5.12 <9.0.0. 
Phpunit since v8.5.12 changed PHP version constraint from ^7.2 to >=7.2 to allow the installation of PHPUnit 8.5 on PHP 8.